doc/fluid/design/dist_train/dist_train_nccl2.md

```diff
@@ -0,0 +1,34 @@
+# Distributed Training with NCCL2
+
+We design a pattern that can enable training with `ParallelExecutor` and
+using NCCL2 as it's collective communication library.
+
+In `ParallelExecutor` we can use `AllReduce` or `Reduce` and `Broadcast`
```

Comment:
> we can use `AllReduce` or `Reduce` and `Broadcast` to do multi GPU training.

Seems only `do variables aggregation between multiple GPU cards` rather than `do multi GPU training` ?

Reply:
broadcasting is not aggregation.